pkg/api/controllers/account_controller.go

```diff
@@ -38,11 +40,40 @@ func (ctl *AccountController) CountAccounts(c *gin.Context) {
 func (ctl *AccountController) GetAccounts(c *gin.Context) {
 	l, _ := c.Get("ledger")
 
+	balance := c.Query("balance")
+	if balance != "" {
+		if _, err := strconv.ParseInt(balance, 10, 64); err != nil {
+			ResponseError(
+				c,
+				ledger.ValidationError{
+					Msg: "invalid parameter 'balance', should be a number"})
+			return
+		}
+	}
+
+	var ok bool
+	var balanceOperator storage.BalanceOperator
+
+	if balanceOperatorStr := c.Query("balance_operator"); balanceOperatorStr != "" {
+		if balanceOperator, ok = storage.NewBalanceOperator(balanceOperatorStr); !ok {
+			ResponseError(
+				c,
+				ledger.ValidationError{
+					Msg: "invalid parameter 'balance_operator', should be one of 'e, gt, gte, lt, lte'"})
+			return
+		}
+	} else {
+		// default value for balanceOperator
+		balanceOperator = storage.BalanceOperatorGte
```

Comment:
Just a minor suggestion :) 
Instead of comment + assignment, a new `const DefaultBalanceOperator = BalanceOperatorGte` in the storage package could be created, and used as the init value line 55.
What do you think?